src/index.js

```diff
@@ -20,6 +20,21 @@ export function isResponseCached(requestKey) {
   return Boolean(responseCache[requestKey]);
 }
 
+export function getCachedResponse(requestKey) {
+  return responseCache[requestKey] || null;
```

Comment:
unsure of desired behavior here, my instinct is to return null but maybe it should throw

Reply:
yeah that makes sense I think

Reply:
just to clarify here to you mean a capital S Symbol -- ie https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol -- or just something to identify a cache miss

For some prior art although it's not exactly the same, apollo will through an error if you try to read from it's cache something that is not there.

Reply:
Thinking about it more, the Symbol seems a little clunky? you'd have to export so they could do equality check? I wonder if throwing an error makes more sense (apollo way, though obviously a much more prescriptive library, though not found in an http sense would or should be an error). Barring that maybe just return undefined since there is no data?